am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +126,41 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * Returns a map of JsonPointers and set of permissions when record with user role and type exists in database.
```

Comment:
I am not a big fun of saying what is returned in the first line of Javadoc. There is dedicated `@return` block to expand on returned value. Javadoc should firstly say what method does not what it returns.

Reply:
Other java docs in this file begin with same pattern and so wanted to keep consistent. Will change now.

Reply:
👍